src/command_modules/azure-cli-resource/tests/recordings/test_list_storage_account.yaml

```diff
@@ -0,0 +1,28 @@
+interactions:
+- request:
```

Comment:
This is the result of you trying out a temporary test case and later removed? Anyway, please remove this recording file which is not associated with any tests